File: test_login.py
import requests

url = "http://localhost:8000/api/auth/login"
payload = {
    "username": "isabel.morocho@unl.edu.ec",
    "password": "password"
}
headers = {
    "Content-Type": "application/json"
}

try:
    response = requests.post(url, json=payload)
    print("Status:", response.status_code)
    print("Response:", response.json())
except Exception as e:
    print(e)

File: test_login.js
const axios = require('axios');
(async () => {
  try {
    const res = await axios.post('http://localhost:8000/api/auth/login', {
      username: 'isabel.morocho@unl.edu.ec',
      password: 'somepassword'
    }, {
      headers: { 'Content-Type': 'application/json' }
    });
    console.log(res.data);
  } catch (e) {
    console.error(e.response ? e.response.data : e.message);
  }
})();
